### DISCUSSION TITLE
Syntax for running a specific seeder

### DISCUSSION THREAD

**Discussion by jmichaelterenin** [tada5hi/typeorm-extension#518] [Q&A]
I'm sorry for what is probably going to sound like a simpleton question, but I'm struggling on the syntax to run a specific seeder:
npm run seed --seed=FeeSeeder

package.json:
...
"seed": "ts-node ./node_modules/typeorm-extension/dist/cli/index.js seed",
...

I've tried adding a ' --' to the end of the command string in package.json (browsed and found this as a solution to adding options), tried --seed FeeSeeder (without the equal sign), and just can't get it to work. Please help

**Comment by tada5hi** [comment]
Does this work for you?
```json
"seed": "ts-node ./node_modules/typeorm-extension/dist/cli/index.js seed --seed FeeSeeder",
```
But i think this is another reason to create an examples directory in this repository ^^.
